app/Http/Requests/StoreTorrentRequest.php

```diff
@@ -105,17 +105,17 @@ function (string $attribute, mixed $value, Closure $fail): void {
             ],
             'description' => [
                 'required',
-                'max:2097152'
+                'max:65535'
             ],
             'mediainfo' => [
                 'nullable',
                 'sometimes',
-                'max:2097152',
+                'max:65535',
             ],
             'bdinfo' => [
                 'nullable',
                 'sometimes',
-                'max:2097152',
```

Comment:
Let's keep the bdinfo one as is so users don't try and paste in 4GB of text. I should have specified earlier but the validation should be less than or equal to the database column, but the limit should still be reasonable.

Reply:
Reverted in 548a7e4d0bfbdf21150557d703cff2606c09c995